packages/ir-sdk/fern/apis/ir-types-latest/definition/http.yml

```diff
@@ -595,6 +596,27 @@ types:
           is the name of the type that contains this property
         type: optional<types.DeclaredTypeName>
 
+  ExampleFileUploadRequestBody:
+    extends: commons.WithJsonExample
+    properties:
+      files: list<ExampleFileProperty>
+      properties: list<ExampleInlinedRequestBodyProperty>
+
+  ExampleFileProperty:
+    properties:
+      name: commons.NameAndWireValue
+      value: ExampleFile
+
+  ExampleFile:
+    union:
+      single: ExampleFileSingle
+      array: list<ExampleFileSingle>
```

Comment:
Do we really need to differentiate from single and array in the IR level? Could we just pass through an array and treat it as single later if has one item?